integration/tools/validation/pom.xml

```diff
@@ -0,0 +1,67 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.alluxio</groupId>
+        <artifactId>alluxio-integration-tools</artifactId>
+        <version>2.3.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>alluxio-integration-tools-validation</artifactId>
+    <packaging>jar</packaging>
+    <name>Alluxio Integration - Validation Tools</name>
+    <description>Tools to validation the underlying environment for Alluxio</description>
+
+    <properties>
+        <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
+        <!-- run properly from sub-project directories -->
+        <build.path>${project.parent.parent.parent.basedir}/build</build.path>
```

Comment:
This is the same as https://github.com/Alluxio/alluxio/blob/b1d5a4e3b8c15eff82f6ffc6c23094d044fb2b3c/integration/tools/hms/pom.xml#L31 but actually i don't think it's correct...